images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -58,25 +60,30 @@ _storage_overrides:
         s3:
           # Intentionally an invalid default name to ensure that this is overridden, and obvious when it's not.
           bucket: "-unconfigured-metering-bucket-name-"
-          awsCredentialsSecretName: ""
+          secretName: ""
           # create buckets by default
           createBucket: true
           region: ""
     reporting-operator:
       spec:
         config:
           aws:
-            secretName: "{{ _storage_spec | json_query('hive.s3.awsCredentialsSecretName') }}"
+            secretName: "{{ _storage_spec | json_query('hive.s3.secretName') }}"
             createSecret: false
```

Comment:
Is _storage_spec or hive.s3 in the values.yaml?

Reply:
`_storage_spec` is a variable in ansible defined above that contains `spec.storage`. This is querying the `spec.storage` section and getting `spec.storage.hive.s3.secretName`

Reply:
👍